<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbjadFormMini</class>
 <widget class="QMainWindow" name="AbjadFormMini">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>159</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sahel</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="accessibleName">
         <string>classFD</string>
        </property>
        <property name="text">
         <string>حساب ابجد:   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="accessibleName">
         <string>classFD</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="accessibleName">
       <string>classFD</string>
      </property>
      <property name="text">
       <string>جزئیات   </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="accessibleName">
       <string>classFD</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
